### PR TITLE
fuzzers: make jobs and workers configurable via environment

### PR DESCRIPTION
**Pull request by tmatth**

### CODE REVIEW

**Comment by tmatth** [comment]
@atoppi this can be specified with `WORKERS=M JOBS=n ./run.sh`

File: fuzzers/run.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@
 set -eu
 
 TARGET=${1-"rtcp_fuzzer"}
+JOBS=${JOBS:-4}
+WORKERS=${WORKERS:-4}
```
Comment (lminiero) [inline fuzzers/run.sh]: I'll let @atoppi comment on the PR in general, but why is the value `-4` there? Shouldn't it be `4`?

Comment (tmatth) [reply fuzzers/run.sh]: @lminiero in bash, that's the syntax to specify default values if missing, it's dash 4, not minus 4.

Comment (lminiero) [reply fuzzers/run.sh]: Thanks for the clarification, I'm an idiot :smile:

Comment (tmatth) [reply fuzzers/run.sh]: @lminiero not at all, I have to look this up every time ;)

**Comment by atoppi** [comment]
@tmatth Thanks for the PR.
I think JOBS and WORKERS should default to a `nproc` related value.

**Comment by tmatth** [comment]
@atoppi updated

File: fuzzers/run.sh [hunk 2]
```diff
@@ -3,8 +3,9 @@
 set -eu
 
 TARGET=${1:-"rtcp_fuzzer"}
-JOBS=${JOBS:-4}
-WORKERS=${WORKERS:-4}
+HALF_NCORES=$(expr $(nproc) / 2)
```
Comment (atoppi) [inline fuzzers/run.sh]: To avoid zero values for single core machines:
```
if [[ $HALF_NCORES -eq 0 ]]; then
 HALF_NCORES=1
fi
```

**Comment by atoppi** [comment]
@tmatth looks good to me, merging.

**Pull request merged**